xml_converter/src/packaging_protobin.cpp

```diff
@@ -239,6 +233,19 @@ void write_protobuf_file(
             std::cout << "Unknown type" << std::endl;
         }
     }
+    return category_to_pois;
+}
+
+void write_protobuf_file(
+    const string& marker_pack_root_directory,
+    const StringHierarchy& category_filter,
+    const map<string, Category>* marker_categories,
+    const vector<Parseable*>* parsed_pois) {
+    std::map<string, std::vector<Parseable*>> category_to_pois;
+    ProtoWriterState state;
+    state.marker_pack_root_directory = marker_pack_root_directory;
+
+    category_to_pois = _construct_category_to_pois_map(parsed_pois);
```

Comment:
You can create an instantiate `categories_to_pois` in one line, no need to split it up across other declarations.

Reply:
I did in another function. I'll make this change.